doc/tutorials/using_windows_as_uos.rst

```diff
@@ -72,43 +72,82 @@ Create a raw disk
-
-   .. note:: Make sure you use GVT-g ``-s 2,pci-gvt -G "$2"`` in the
-      ``acrn-dm`` command line. Currently, we do not support creating a
-      windows image by GVT-d.
-
-   - Change ``-s 3,virtio-blk,./win10-ltsc.img`` to your path to the Windows
-     10 image.
+#. Refer to :ref:`gpu-passthrough` to enable GVT-d GOP feature. Then
+   copy above ``.iso`` files and the build ``OVMF.fd`` to
+   ``/home/acrn/work``
+#. Prepare WaaG install script
+
+.. code-block:: bash
+
+   vim /home/acrn/work/install_win.sh
+
+   #!/bin/bash
+   function launch_win()
+   {
+   vm_name=win_vm$1
+   #check if the vm is running or not
+   vm_ps=$(pgrep -a -f acrn-dm)
+   result=$(echo $vm_ps | grep "${vm_name}")
+   if [[ "$result" != "" ]]; then
+     echo "$vm_name is running, can't create twice!"
+     exit
+   fi
+   echo "8086 9ded" > /sys/bus/pci/drivers/pci-stub/new_id
+   echo "0000:00:14.0" > /sys/bus/pci/devices/0000:00:14.0/driver/unbind
+   echo "0000:00:14.0" > /sys/bus/pci/drivers/pci-stub/bind
+   echo "8086 3ea0" > /sys/bus/pci/drivers/pci-stub/new_id
+   echo "0000:00:02.0" > /sys/bus/pci/devices/0000:00:02.0/driver/unbind
+   echo "0000:00:02.0" > /sys/bus/pci/drivers/pci-stub/bind
+   #for memsize setting
+   mem_size=4096M
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
+     -s 2,passthru,0/2/0,gpu \
+     -s 8,virtio-net,tap0 \
+     -s 4,virtio-blk,/home/acrn/work/win10-ltsc.img
+     -s 5,ahci,cd:./home/acrn/work/windows.iso \
+     -s 6,ahci,cd:./home/acrn/work/winvirtio.iso \
+     -s 7,passthru,0/14/0,d3hot_reset \
+     --ovmf ./home/acrn/work/OVMF.fd \
+     --windows \
+     $vm_name
+   }
+   # offline SOS CPUs except BSP before launch UOS
+   for i in `ls -d /sys/devices/system/cpu/cpu[1-99]`; do
+           online=`cat $i/online`
+           idx=`echo $i | tr -cd "[1-99]"`
+           echo cpu$idx online=$online
+           if [ "$online" = "1" ]; then
+                   echo 0 > $i/online
+                   # during boot time, cpu hotplug may be disabled by pci_device_probe during a pci module insmod
+                  while [ "$online" = "1" ]; do
+                           sleep 1
+                           echo 0 > $i/online
+                           online=`cat $i/online`
+                   done
+                   echo $idx > /sys/class/vhm/acrn_vhm/offline_cpu
+           fi
+   done
+   launch_win 1 "64 448 8"
+
+Install Windows 10 by GVT-d
+---------------------------
 
-   - Add ``-s 6,xhci,1-5:1-9``. You may need to change 1-5:1-9 to match the
-     ports of the USB keyboard/mouse and flash on your platform.
 
-   - Add ``-s 8,ahci,cd:./windows10-LTSC-17763.iso`` to point to the Win10
-     ISO.
+#. Run **install_win.sh**
 
-   - Add ``-s 9,ahci,cd:./winvirtio.iso`` to point to your path to the
-     winvirtio iso.
+   .. code-block:: bash
 
-Install Windows 10 by GVT-g
----------------------------
+      cd /home/acrn/work/
```

Comment:
Do we not need to make the file executable? (i.e. `chmod +x install_win.sh`)

Reply:
Done!